docs/source/tutorial/using_wandb.rst

```diff
@@ -0,0 +1,64 @@
+Using Weights and Biases
+========================
+`Weights and Biases <http://wandb.ai/>`_ (WANDB) is a service for tracking experimental results and various artifacts
+appearing whn training ML models.
+
+
+After `registering <https://app.wandb.ai/login?signup=true>`_ for WANDB, do the following:
+
+1. Create a project in WANDB, for example, with the name ``pykeen_project`` at
+   ``https://app.wandb.ai/<your username>/new``
+2. Install WANDB on your machine with ``pip install wandb``
+3. Setup your computer for use with WANDB by using either of the following two instructions from
+   https://github.com/wandb/client#running-your-script:
+
+   1. Navigate to https://app.wandb.ai/settings, copy your API key, and set the ``WANDB_API_KEY`` environment variable
+   2. Interactively run ``wandb login``
+
+Now you can simply specify this project name when initializing a pipeline, and everything else will work automatically!
+
+Pipeline Example
+----------------
+This example shows using WANDB with the :func:`pykeen.pipeline.pipeline` function.
+
+.. code-block:: python
+
+    from pykeen.pipeline import pipeline
+
+    results = pipeline(
+        model='RotatE',
+        dataset='Kinships',
+        result_tracker='wandb',
+        result_tracker_kwargs=dict(
+            project='pykeen_project',
+        ),
+    )
+
+
+You can navigate to the created project in WANDB and observe a running experiment.
+Further tweaking of appearance, charts, and other settings is described in the official
+`documentation <https://docs.wandb.com/>`_
+
+HPO Example
+-----------
+This example shows using WANDB with the :func:`pykeen.hpo.hpo_pipeline` function.
+
+.. code-block:: python
+
+    from pykeen.hpo import hpo_pipeline
+
+    results = hpo_pipeline(
+        model='RotatE',
+        dataset='Kinships',
+        result_tracker='wandb',
+        result_tracker_kwargs=dict(
+            project='pykeen_project',
+            experiment='new run',
```

Comment:
All pipelines will be passed the same experiment. What happens? Do they get overwritten?

Reply:
No, there will be N runs with the same name but different hashes (unique ids). But generally `experiment` is advisable to use for single runs, not for batches of multiple runs.